num_tower: 4
LED_per_tower: 4
